xdp-forward/xdp-forward.c

```diff
@@ -81,17 +81,18 @@ static int find_prog(struct iface *iface, bool detach)
 }
 
 struct load_opts {
-	enum fib_mode fib_mode;
+	enum fwd_mode fwd_mode;
 	enum xdp_attach_mode xdp_mode;
 	struct iface *ifaces;
-} defaults_load = { .fib_mode = FIB_MODE_FULL };
+} defaults_load = { .fwd_mode = FIB_MODE_FULL };
 
 struct prog_option load_options[] = {
-	DEFINE_OPTION("fib-mode", OPT_ENUM, struct load_opts, fib_mode,
+	DEFINE_OPTION("fwd-mode", OPT_ENUM, struct load_opts, fwd_mode,
 		      .short_opt = 'f',
-		      .typearg = fib_modes,
+		      .typearg = fwd_modes,
 		      .metavar = "<direct|full>",
-		      .help = "Perform full or direct (skipping rules) FIB lookups; default full"),
+		      .help = "Exec xdp-redirect performing lookup on the FIB table (full or direct); "
+		      "fib-direct skips fib-rules. Default value is fib-full"),
```

Comment:
The metavar should be updated with the new names as well

Reply:
@tohojo this should be updated.